<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>2000</width><height>1000</height></rect>
  </property>
  <property name="windowTitle">
   <string>3d computer vision framework</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>

  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator{width:15px;height:15px;}
QRadioButton::indicator{width:15px;height:15px;}</string>
   </property>

   <layout class="QHBoxLayout" name="mainLayout">
    <property name="spacing"><number>6</number></property>
    <property name="margin"><number>11</number></property>

    <item>
     <widget class="QWidget" name="sidePanel">
      <property name="maximumWidth"><number>260</number></property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch><verstretch>0</verstretch>
       </sizepolicy>
      </property>

      <layout class="QVBoxLayout" name="sideLayout">
       <item><widget class="Line" name="line_3">
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
       </widget></item>

       <item><widget class="QPushButton" name="pushButton">
         <property name="text"><string>Open PLY</string></property>
       </widget></item>

       <item><widget class="Line" name="line">
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
       </widget></item>

       <item><widget class="QLabel" name="label_2">
         <property name="text"><string>Point size [1,10]:</string></property>
       </widget></item>

       <item><widget class="QSlider" name="horizontalSlider">
         <property name="minimum"><number>1</number></property>
         <property name="maximum"><number>10</number></property>
         <property name="value"><number>3</number></property>
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
       </widget></item>

       <item><widget class="Line" name="line_depth">
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
       </widget></item>

       <item><widget class="QLabel" name="label_depth">
         <property name="text"><string>kd-tree depth [1,10]:</string></property>
       </widget></item>

       <item><widget class="QSlider" name="horizontalSliderDepth">
         <property name="minimum"><number>1</number></property>
         <property name="maximum"><number>10</number></property>
         <property name="value"><number>3</number></property>
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
       </widget></item>

       <item><spacer name="verticalSpacer">
         <property name="orientation"><enum>Qt::Vertical</enum></property>
         <property name="sizeHint" stdset="0">
          <size><width>20</width><height>40</height></size>
         </property>
       </spacer></item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="GLWidget" name="glwidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch><verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="canvasLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>

 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
